src/project_creator/templates/project/{{cookiecutter.name}}/doc/components/index.rst

```diff
@@ -0,0 +1,23 @@
+Components
```

Comment:
Variant report includes all the rst files relevant for the components. This is required to get traceability between all sphinx ids. Without this, one can not trace SW requirements (which are in the project doc) to SW detail design (which are in the components index.rst)